test-init.gradle

```diff
@@ -0,0 +1,45 @@
+initscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath files(
+            '/root/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bc-fips/2.0.0/ee9ac432cf08f9a9ebee35d7cf8a45f94959a7ab/bc-fips-2.0.0.jar',
```

Comment:
🤔 what happens if we don't have them in the cache?

Reply:
Good point. this was a hack I added early on. I was hoping to avoid it or make it more dynamic. I'll continue to work at this, what is in this iteration here is unacceptable.